app/javascript/admin/controllers/creator_settings_controller.js

```diff
@@ -3,7 +3,7 @@ import { isLowContrast } from '@utilities/color/contrastValidator';
 import { brightness } from '@utilities/color/accentCalculator';
 
 const MAX_LOGO_PREVIEW_HEIGHT = 80;
-const MAX_LOGO_PREVIEW_WIDTH = 220;
+const MAX_LOGO_PREVIEW_WIDTH = 200;
```

Comment:
@nickytonline can I ask how we determine the number?

Reply:
This is what is set for the `.site-logo` class. But now that you mention it, I can load the value from the CSS var so it remains consistent without having to change this code later.